.sync.yml

```diff
@@ -1,13 +1,69 @@
 ---
 appveyor.yml:
-  delete: true
-spec/spec_helper.rb:
-  allow_deprecations: true
+  environment:
+  PUPPET_GEM_VERSION: "~> 4.0"
+  matrix:
+    - RUBY_VERSION: 24-x64
+      CHECK: "syntax lint"
+    - RUBY_VERSION: 24-x64
+      CHECK: metadata_lint
+    - RUBY_VERSION: 24-x64
+      CHECK: rubocop
+
 .travis.yml:
+  bundle_args: --without system_tests
+  docker_sets:
+    - set: docker/centos-7
+      options:
+    - set: docker/ubuntu-14.04
+      options:
+  docker_defaults:
+    bundler_args: ""
+  secure: ""
+  branches:
+    - release
   extras:
-  - rvm: 2.1.9
-    script: bundle exec rake rubocop
+    - rvm: 2.1.9
+      script: "\"bundle exec rake release_checks\""
+
+Gemfile:
+  required:
+    ':system_tests':
+      - gem: 'puppet-module-posix-system-r#{minor_version}'
+        platforms: ruby
+      - gem: 'puppet-module-win-system-r#{minor_version}'
+        platforms:
+          - mswin
+          - mingw
+          - x64_mingw
+      - gem: beaker
+        version: '~> 3.13'
+        from_env: BEAKER_VERSION
+      - gem: beaker-abs
+        from_env: BEAKER_ABS_VERSION
+        version: '~> 0.1'
+      - gem: beaker-pe
+      - gem: beaker-hostgenerator
+        from_env: BEAKER_HOSTGENERATOR_VERSION
+      - gem: beaker-rspec
+        from_env: BEAKER_RSPEC_VERSION
+    ':development':
+      - gem: puppet-blacksmith
+        version: '~> 3.4'
+      - gem: puppet-lint-i18n
+
+Rakefile:
+  requires:
+    - puppet_blacksmith/rake_tasks
+    - puppet_pot_generator/rake_tasks
+
+spec/spec_helper.rb:
+  spec_overrides:
+    -  "require 'spec_helper_local'"
+
 .rubocop.yml:
-  unmanaged: true
-.rubocop_todo.yml:
-  unmanaged: true
+   default_configs:
+    inherit_from: .rubocop_todo.yml
+    require:
```

Comment:
By adding the `rubocop-rspec` require here it allows us to require multiple things. If I only require `rubocop-i18n` the `rubocop-rspec` from the template it overwritten.

Reply:
`rubocop-i18n` is required because this module has been internationalized.